src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -268,8 +269,7 @@ export const BaseArtworkFilter: React.FC<
 
   return (
     <Box mt={tokens.mt} {...rest}>
-      <Box id="jump--artworkFilter" />
-
+      <ArtworkFilterAnchor />
```

Comment:
if this going to be added in a follow-up PR, can we find other instances of this kind of use elsewhere in app and replace with `ArtworkFilterAnchor`?